model_databank/utils.py

```diff
@@ -28,3 +32,19 @@ def zip_model_files(model_reference):
     zip_dir(repo_path, zip_file)
     zip_file.close()
     return zip_file_full_path, latest_revision
+
+
+def get_organisation_ids_by_user_permission(user, permission):
+    """
+    Return the organisation ids the user has the permission for.
+
+    :raise - requests.exceptions.HttpError when status code is not ok (2xx)
+    :return - list of organisation ids
+
+    """
+    permissions_api_url = '{}{}/{}/'.format(
+        settings.PERMISSIONS_API_URL, user.username, permission)
```

Comment:
Goed punt, hij staat in de settings van het overkoepelend project: ``threedi-msr``. Ik zal hier morgen naar kijken.